src/graphcast_agent/message_typing.rs

```diff
@@ -136,13 +136,16 @@ impl<T: Message + ethers::types::transaction::eip712::Eip712 + Default + Clone +
         &self,
         node_handle: &WakuNodeHandle<Running>,
         pubsub_topic: WakuPubSubTopic,
-        content_topic: WakuContentTopic,
+        content_topic: &str,
     ) -> Result<String, WakuHandlingError> {
         let mut buff = Vec::new();
         Message::encode(self, &mut buff).expect("Could not encode :(");
 
+        let waku_content_topic = WakuContentTopic::from_str(content_topic)
```

Comment:
why should content topic be made inside of sending the message?

Reply:
for the sake of the mocks, in order to construct a custom `nonce` and other fields and to call the `send_to_waku` directly, since the `send_message` method is quite limiting. I guess the same could be achieve by some sort of helper that accepts all fields and then just bolts the topic on, do you think that would be better?

Reply:
Do you think it's possible to make content topic within the mocked tests and pass it into `send_to_waku`?

Reply:
it is, we just need to re-export `WakuContentTopic` from the sdk, will do that